tests/unit/featurizers/test_atom.py

```diff
@@ -5,7 +5,7 @@
 from rdkit import Chem
 from rdkit.Chem.rdchem import HybridizationType
 
-from chemprop.v2.featurizers.multihot import AtomFeaturizer
+from chemprop.v2.featurizers import AtomFeaturizer
```

Comment:
I think the changes in this file should be in `chemprop/v2/tests/unit/featurizers/test_atom.py` and I think @am2145 addresses most of the changes in this commit in his PR https://github.com/chemprop/chemprop/pull/463. It might make sense to rebase from his once his is merged.